cc3d/player5/Plugins/ViewManagerPlugins/ScreenshotManager.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, _tabViewWidget):
         self.screenshot_number_of_digits = len(str(self.basicSimulationData.numberOfSteps))
 
         # we limit max number of screenshots to discourage users from using screenshots as their main analysis tool
-        self.maxNumberOfScreenshots = 30
+        self.maxNumberOfScreenshots = 100
```

Comment:
I am not sure this is needed. We have a movie maker that relies on screen shots as frames, so it should be up to the user to make as many screen shots as they like. It is discourteous to warn the user only *after* the limit has been reached that such a limit exists.

Reply:
The name of the variable might be confusing. maxNumberOfScreenshots is the number of screenshot configurations, or number of entries in the screenshots.json  file. It is not the number of actual screenshots. This limit is really high and we can increase if needed. The reason I put it here is to avoid the situation where users will create so many screenshot configurations that 99% of simulation time will be spent taking screenshots. It is not a big deal to lift this limitation though bu

Reply:
Ohhh, it is the maximum number of camera angles for the screenshots data dict, not necessarily the exact number of screenshots. That's fine.

Reply:
yes it is.